src/M6Web/Bundle/ElasticsearchBundle/Connection/CurlMultiConnectionDecorator.php

```diff
@@ -45,11 +45,14 @@ public function performRequest($method, $uri, $params = null, $body = null, $opt
             throw $e;
         }
 
+        $took = $this->extractTookFromResponse($response);
+
         $this->dispatchEvent(
             $method,
             $uri,
-            $response['requestInfo']['http_code'],
-            $response['requestInfo']['total_time'] * 1000  // Convert from seconds to milliseconds
+            $response['status'],
```

Comment:
Are you sure about the modification of `$response['requestInfo']['http_code']` to `$response['status']` ? 

I thought that curl_multi was not setting a `status` directly on the `$response`.